Releasenotes.md

```diff
@@ -1,6 +1,76 @@
 # Release notes BRK-Bevragen
 
-## Versie 1.1.1:
+## **Versie 1.2.0:**
+
+### openapi.yaml
+
+- Algemeen
+  - Diverse verbeteringen in de documentatie (descriptions) en examples.
+  - Er is gebruikgemaakt van Haal-Centraal-common versie 1.2.0 (was 1.0.0)
+  - De default-waarde voor de Accept-Crs header is verwijderd.
+  - Uit alle componentnamen zijn de underscores verwijderd.
+
+
+- KadastraalOnroerendeZaak
+  - De default-waarde voor de Accept-Crs header is verwijderd bij /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}
+  - PrivaatrechtelijkeBeperkingen, hypotheken en beslagen zijn toegevoegd aan de _embedded en _links. De bijbehorende identificaties zijn opgenomen in de KadastraalOnroerendeZaak.
+  - LocatieKadastraalObject levert naast nummeraanduidingIdentificatie nu ook het adres.
+
+
+- Zakelijkgerechtigden
+  - Accept-Crs header en Content-Crs header zijn toegevoegd bij
+    - /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}/zakelijkgerechtigden/{zakelijkgerechtigdeidentificatie}
+    - /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}/zakelijkgerechtigden
+
+- Aantekening  
+  - Properties begrenzing en indicatieOorspronkelijkObject worden niet meer geleverd en zijn deprecated verklaard
+
+
+- KadasterNatuurlijkPersoon
+  - Aanschrijfwijze, aanhef en gebruikInLopendeTekst zijn toegevoegd.
+
+
+- Hypotheken
```

Comment:
is het niet handiger eerst alle wijzigingen op endpoints te benoemen, en daarna alle (detail) wijzigingen op componenten?

Reply:
Aangepast